cmd/dqlite-demo/dqlite-demo.go

```diff
@@ -125,7 +127,8 @@ Complete documentation is available at https://github.com/canonical/go-dqlite`,
 }
 
 const (
-	schema = "CREATE TABLE IF NOT EXISTS model (key TEXT, value TEXT, UNIQUE(key))"
-	query  = "SELECT value FROM model WHERE key = ?"
-	update = "INSERT OR REPLACE INTO model(key, value) VALUES(?, ?)"
+	schema          = "CREATE TABLE IF NOT EXISTS model (key TEXT, value TEXT, UNIQUE(key))"
```

Comment:
Was this done by go fmt? If not then we should avoid custom formatting.

Reply:
Normally `go-fmt` runs when I save a file, I'll doublecheck to be sure though.